package-lock.json

```diff
@@ -40,7 +40,6 @@
 				"minimist": "^1.2.8",
 				"moo": "^0.5.2",
 				"node-retrieve-globals": "^6.0.0",
-				"normalize-path": "^3.0.0",
```

Comment:
Does this change belong here? Though I’m on board with it—I don’t think it matters as it is used downstream by `@11ty/eleventy-utils` and will be installed anyway.

Reply:
I’m going to go ahead and push this dep removal independently—ignore my comment above!